build.gradle

```diff
@@ -40,9 +40,9 @@ try {
 }
 
 project.ext.minSdkVersion = 16
-project.ext.compileSdkVersion = 27
-project.ext.buildToolsVersion = '27.0.3'
-project.ext.supportLibVersion = '27.1.1'
+project.ext.compileSdkVersion = 28
+project.ext.buildToolsVersion = '28.0.3'
```

Comment:
https://developer.android.com/studio/releases/gradle-plugin#3-1-0, the Build Tools Version no longer needs to be specified. This property can be removed.